be/src/storage/tablet_manager.cpp

```diff
@@ -1028,83 +1091,53 @@ Status TabletManager::start_trash_sweep() {
     }
 
     std::vector<DroppedTabletInfo> tablets_to_check;
+    std::vector<DroppedTabletInfo> tablets_redundant_to_check;
```

Comment:
use `map`？

Reply:
I has been optimized here

Reply:
hi @wyb @meegoo @sevev 
please reveiw the code